openlibrary/templates/lib/nav_foot.html

```diff
@@ -43,6 +42,7 @@ <h2>$:_('Help')</h2>
           <li><a href="/help">$_('Help Center')</a></li>
           <li><a href="/contact?$urlencode(dict(path=request.fullpath))" title="$_('Problems')">$_('Report A Problem')</a></li>
           <li><a href="/help/faq/editing" title="$_('Suggest Edits')">$_('Suggesting Edits')</a></li>
+	  <li><a href="/books/add" title="$_('Add a new book to Open Library')">$_('Add a Book')</a></li>
```

Comment:
```suggestion
          <li><a href="/books/add" title="$_('Add a new book to Open Library')">$_('Add a Book')</a></li>
```

Please ensure this line aligns with the others.

Reply:
@scottbarnes Thanks for the review. 
So I think that you want me to rename the 'Add a Book' label to 'Add a new book to open library' in the help section?

Reply:
I'm sorry for not being clear. The [issue](#8213) still has the label `Needs: Community Discussion`, and I can't quite tell if that's an oversight or if there is interest in further discussion.

In terms of the my code review, I just meant to suggest further indenting the line you moved so that each line has the same amount of indentation.

Reply:
Hi @scottbarnes I have made the changes suggested by you.
I think now we have to wait for Tuesday's community discussion to see if this [PR](https://github.com/internetarchive/openlibrary/pull/8394) can be merged.

Reply:
Thanks, @piyush-poddar. This looks good to me and I don't think there is anything further for you to do. We'll verify on Tuesday if there is anything further, but this PR addresses the issue as far as I can see. :)

Reply:
Thanks for your assistance @scottbarnes